src/testing/flow_solver_cases/flow_solver_case_base.cpp

```diff
@@ -7,10 +7,11 @@ template<int dim, int nstate>
 FlowSolverCaseBase<dim, nstate>::FlowSolverCaseBase(const PHiLiP::Parameters::AllParameters *const parameters_input)
         : all_param(*parameters_input)
         , mpi_communicator(MPI_COMM_WORLD)
-        , mpi_rank (dealii::Utilities::MPI::this_mpi_process(MPI_COMM_WORLD))
-        , n_mpi (dealii::Utilities::MPI::n_mpi_processes(MPI_COMM_WORLD))
+        , mpi_rank(dealii::Utilities::MPI::this_mpi_process(MPI_COMM_WORLD))
+        , n_mpi(dealii::Utilities::MPI::n_mpi_processes(MPI_COMM_WORLD))
         , pcout(std::cout, mpi_rank==0)
-          {}
+        , unsteady_data_table_filename_with_extension(all_param.flow_solver_param.unsteady_data_table_filename+".txt")
```

Comment:
Why move this to flow solver case base? Not all flow solver cases will necessarily require it.

Reply:
Maybe we should just replace `sensitivity_table_filename` and `unsteady_data_table_file_name` by `data_table_filename`? This way we don't have case specific variables in the flow solver case base class

Reply:
I would agree except that I need both, since one of my tests is outputting the solution and a sensitivity. Thats why I needed to create a second filename parameter.

Reply:
Actually, you do use it in `compute_unsteady_data_and_write_to_table` in both your `BurgersRewienskiSnapshot` and `BurgersViscousSnapshot` classes so I think we should leave it -- but can consider renaming the variable it if you want

Reply:
I was more looking at it from the point of view that not every future test will necessarily be computing unsteady data and writing to a table so it probably doesn't make sense to have a such a specific variable in the flow solver case base class. Renaming it to something more generic would probably make more sense

Reply:
I've addressed this in the most recent commit, I simply removed it from `FlowSolverCaseBase` and put it in `PeriodicCubeFlow`; and restored your Burgers classes how they originally were